backend/app/test/profileTest.ts

```diff
@@ -0,0 +1,137 @@
+import fs = require('fs');
+import chai = require('chai')
+import chaiHttp = require('chai-http')
+import assert = require('assert')
+
+chai.use(chaiHttp)
+
+describe('Testing the different components used on the profile page', () => {
+  const agentAdmin = chai.request.agent('http://localhost:8000/api')
+  const agentUser = chai.request.agent('http://localhost:8000/api')
+  let currentAvatar = ''
+  let currentBanner = ''
+
+  let amountOfFollowers = 0
+
+  before((done) => {
+    agentUser
+      .post('/login')
+      .send({
+        username: 'User',
+        password: 'Qwerty123'
+      })
+      .end(() => {
+        agentAdmin
+          .post('/login')
+          .send({
+            username: 'Admin',
+            password: 'Qwerty123'
+          })
+          .end(() => {
+            done()
+          })
+      })
+  })
+
+  it('Should update user', (done) => {
+    agentUser
+      .put('/profile/User')
+      .send({
+        username: 'User',
+        description: 'update user'
+      })
+      .end((err, res) => {
+        assert.equal(res.status, 200)
+      })
+
+    agentUser
+      .get('/profile/User')
+      .end((err, res) => {
+        assert.equal(res.body.profile.description, 'update user')
+        done()
+      })
+  })
+
+  it('Should retrieve the current amount of followers', (done) => {
```

Comment:
```suggestion
  it('Should return follower amount of user', (done) => {
```

Why retrieve instead of return?